### PR TITLE
header was created

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](vanyachyzh.github.io/layout_moyo-header/)
- [TEST REPORT LINK](vanyachyzh.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Comment by OlEzhA0** [comment]
Preview links unavailable, please don't remove `https://` from the begging of the link in the Readme.md file

File: src/style.css [hunk 1]
```diff
@@ -1 +1,89 @@
...
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  top: 56px;
+  content: " ";
+  background-color: #00acdc;
```
Comment (OlEzhA0) [inline src/style.css:84]: move repeating color to the variable

File: src/style.css [hunk 2]
```diff
@@ -1 +1,89 @@
...
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  top: 56px;
```
Comment (OlEzhA0) [inline src/style.css:82]:
```suggestion
  bottom: 0;
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,89 @@
...
+  content: " ";
+  background-color: #00acdc;
+  display: block;
```
Comment (OlEzhA0) [inline src/style.css:85]:
```suggestion
  left: 0;
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
Hey! Please check my comments

File: src/index.html [hunk 4]
```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      
+      <a class="img-link" href="#home">
```
Comment (liliia-livandovskaya) [inline src/index.html:15]:
```suggestion
    <header class="header">
      <a class="img-link" href="#home">
```

File: src/index.html [hunk 5]
```diff
@@ -9,6 +9,53 @@
...
+      </a>
+      
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a href="#" class="is-active link">Apple</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="link">Samsung</a>
+          </li>
```
Comment (liliia-livandovskaya) [inline src/index.html:30]:
```suggestion
          <li class="nav-list__item">
            <a href="#" class="is-active link">Apple</a>
          </li>
          
          <li class="nav-list__item">
            <a href="#" class="link">Samsung</a>
          </li>
```

**Review by liliia-livandovskaya — APPROVED** [review]
Great job! 👍 Pay attention to the code style

**Pull request closed without merging**